manim/scene/scene.py

```diff
@@ -28,28 +28,34 @@
 class Scene(Container):
     """A Scene is the canvas of your animation.
 
-    All of your own named Scenes will be subclasses of Scene, or other named
-    scenes.
+    The primary role of :class:`Scene` is to provide the user with tools to manage
+    mobjects and animations.  Generally speaking, a manim script consists of a class
+    that derives from :class:`Scene` whose :meth:`Scene.construct` method is overriden
+    by the user's code.
+
+    Mobjects are displayed on screen by calling :meth:`Scene.add` and removed from
+    screen by calling :meth:`Scene.remove`.  All mobjects currently on screen are kept
+    in :attr:`Scene.mobjects`.  Animations are played by calling :meth:`Scene.play`.
+
+    A :class:`Scene` is rendered internally by calling :meth:`Scene.render`.  This in
+    turn calls :meth:`Scene.setup`, :meth:`Scene.construct`, and
+    :meth:`Scene.tear_down`, in that order.
+
+    It is not recommended to override the ``__init__`` method in user Scenes.  For code
```

Comment:
I don't understand this sentence, which __init__ function should be overwritten?
Can we maybe reference this to an example?

Reply:
What I tried to say is that if the user creates a class that inherits from Scene, like `MyScene(Scene)`, they should try not to override the constructor method.

I don't know what kind of example this would link to, as it is mentioning something that _should not_ be done.

Reply:
Okay, now I get it.
Maybe we can have also a section in the docs which is titled "Best Practice and things to avoid"?

Reply:
That makes sense, I can see a tutorial including that. But that doesn't sound like part of this PR.